src/components/sortableList/SortableListItem.tsx

```diff
@@ -100,7 +103,12 @@ const SortableListItem = (props: Props) => {
           newIndex = Math.sign(newIndex - oldIndex) + oldIndex;
         }
 
-        const itemIdToSwap = getIdByItemIndex(itemsOrder.value, newIndex);
+        let itemIdToSwap = getIdByItemIndex(itemsOrder.value, newIndex);
+
+        while (inertIds.value[itemIdToSwap]) {
```

Comment:
Why using `if` here doesn't work? 
I tried it myself and I saw the bug, but won't you expect it to work with an `if` as well? 
I mean, why do we need the `while` here?

Reply:
Not sure what the bug you saw, the while is if you put two consecutive locked items

Reply:
Hmm ok, I think I understand now. 
What will happen of the last item is a locked one and ill try to drag over it? wouldn't we get stuck in the loop?
Are there other cases that might cause us to stuck in a loop?

Reply:
No, I think it's fine if first and last items don't create issue 👌